---
layout: page
title: Publications
permalink: /publicatons/
---

\[[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/?term=welch%2C+joshua+d%5Bauthor%5D)\] \[[Google Scholar](https://scholar.google.com/citations?user=XQ7bqCMAAAAJ&hl=en)\]

Kriebal AR, Welch JD. Nonnegative matrix factorization integrates single-cell multi-omic datasets with partially overlapping features. bioRxiv. \[[HTML](https://www.biorxiv.org/content/10.1101/2021.04.09.439160v1)

Matsushita Y, Ka Yan Chu A, Ono W, Welch JD, Ono N. Intercellular interactions of an adipogenic CXCL12‐expressing stromal cell subset in murine bone marrow. Journal of Bone and Mineral Research 2021. https://doi.org/10.1002/jbmr.4282 \[[HTML](https://asbmr.onlinelibrary.wiley.com/doi/10.1002/jbmr.4282)\]

Yu H, Welch JD. Predicting unobserved cell states from disentangled representations of single-cell data using generative adversarial networks. Learning Meaningful Representations of Life (LMRL 2020), Workshop at NeurIPS 2020. https://www.biorxiv.org/content/10.1101/2021.01.15.426872v1

Bao Y, Welch JD. Real-Time, Direct Classification of Nanopore Signals Using SquiggleNet. Machine Learning in Computational Biology (MLCB 2020). https://www.biorxiv.org/content/10.1101/2021.01.15.426907v1

BRAIN Initiative Cell Census Network (BICCN), Ricky S. Adkins, Andrew I. Aldridge, Shona Allen, Seth A. Ament, Xu An, Ethan Armand, Giorgio A. Ascoli, Trygve E. Bakken, Anita Bandrowski, Samik Banerjee, Nikolaos Barkas, Anna Bartlett, Helen S. Bateup, M. Margarita Behrens, Philipp Berens, Jim Berg, Matteo Bernabucci, Yves Bernaerts, Darren Bertagnolli, Tommaso Biancalani, Lara Boggeman, A. Sina Booeshaghi, Ian Bowman, Héctor Corrada Bravo, Cathryn René Cadwell, Edward M. Callaway, Benjamin Carlin, Carolyn O'Connor, Robert Carter, Tamara Casper, Rosa G. Castanon, Jesus Ramon Castro, Rebecca K. Chance, Apaala Chatterjee, Huaming Chen, Jerold Chun, Carlo Colantuoni, Jonathan Crabtree, Heather Creasy, Kirsten Crichton, Megan Crow, Florence D. D'Orazi, Tanya L. Daigle, Rachel Dalley, Nick Dee, Kylee Degatano, Benjamin Dichter, Dinh Diep, Liya Ding, Song-Lin Ding, Bertha Dominguez, Hong-Wei Dong, Weixiu Dong, Elizabeth L. Dougherty, Sandrine Dudoit,  Joseph R. Ecker,  Stephen W. Eichhorn, Rongxin Fang, Victor Felix, Guoping Feng, Zhao Feng, Stephan Fischer, Conor Fitzpatrick, Olivia Fong, Nicholas N. Foster, William Galbavy, James C. Gee, Satrajit S. Ghosh, Michelle Giglio, Thomas H. Gillespie, Jesse Gillis, Melissa Goldman, Jeff Goldy, Hui Gong, Lin Gou, Michael Grauer, Yaroslav O. Halchenko, Julie A. Harris, Leonard Hartmanis, Joshua T. Hatfield, Mike Hawrylycz, Brian Helba, Brian R. Herb, Ronna Hertzano, Houri Hintiryan, Karla E. Hirokawa, Dirk Hockemeyer, Rebecca D. Hodge, Greg Hood, Gregory D. Horwitz, Xiaomeng Hou, Lijuan Hu, Qiwen Hu, Z. Josh Huang, Bingxing Huo, Tony Ito-Cole, Matthew Jacobs, Xueyan Jia, Shengdian Jiang, Tao Jiang, Xiaolong Jiang, Xin Jin, Nikolas L. Jorstad, Brian E. Kalmbach, Jayaram Kancherla, C. Dirk Keene, Kathleen Kelly, Farzaneh Khajouei, Peter V. Kharchenko, Gukhan Kim, Andrew L. Ko, Dmitry Kobak, Kishori Konwar, Daniel J. Kramer, Fenna M. Krienen, Matthew Kroll, Xiuli Kuang, Hsien-Chi Kuo, Blue B. Lake, Rachael Larsen, Kanan Lathia, Sophie Laturnus, Angus Y. Lee, Cheng-Ta Lee, Kuo-Fen Lee, Ed S. Lein, Phil Lesnar, Anan Li, Xiangning Li, Xu Li, Yang Eric Li, Yaoyao Li, Yuanyuan Li, Byungkook Lim, Sten Linnarsson, Christine S. Liu, Hanqing Liu, Lijuan Liu, Jacinta D. Lucero, Chongyuan Luo, Qingming Luo, Evan Z. Macosko, Anup Mahurkar, Maryann E. Martone, Katherine S. Matho, Steven A. McCarroll, Carrie McCracken, Delissa McMillen, Elanine Miranda, Partha P Mitra, Paula Assakura Miyazaki, Judith Mizrachi, Stephanie Mok, Eran A. Mukamel, Shalaka Mulherkar, Naeem M. Nadaf, Maitham Naeemi, Arun Narasimhan, Joseph R. Nery, Lydia Ng, John Ngai, Thuc Nghi Nguyen, Lance Nickel, Philip R. Nicovich, Sheng-Yong Niu, Vasilis Ntranos, Michael Nunn, Dustin Olley, Joshua Orvis, Julia K. Osteen, Pavel Osten, Scott F. Owen, Lior Pachter, Ramesh Palaniswamy, Carter R. Palmer, Yan Pang, Hanchuan Peng, Thanh Pham, Antonio Pinto-Duarte, Nongluk Plongthongkum, Olivier Poirion, Sebastian Preissl, Elizabeth Purdom, Lei Qu, Mohammad Rashid, Nora M. Reed, Aviv Regev, Bing Ren, Miao Ren, Christine Rimorin, Davide Risso, Angeline C. Rivkin, Rodrigo Muñoz-Castañeda, William J. Romanow, Alexander J. Ropelewski, Hector Roux de Bézieux, Zongcai Ruan, Rickard Sandberg, Steven Savoia, Federico Scala, Michael Schor, Elise Shen, Kimberly Siletti, Jared B. Smith, Kimberly Smith, Saroja Somasundaram, Yuanyuan Song, Staci A. Sorensen, David A. Stafford, Kelly Street, Josef Sulc, Susan Sunkin, Valentine Svensson, Pengcheng Tan, Zheng Huan Tan, Bosiljka Tasic, Carol Thompson, Wei Tian, Timothy L. Tickle, Michael Tieu, Jonathan T. Ting, Andreas Savas Tolias, Amy Torkelson, Herman Tung, Eeshit Dhaval Vaishnav, Koen Van den Berge, Cindy T.J. van Velthoven, Charles R. Vanderburg, Matthew B. Veldman, Minh Vu, Wayne Wakeman, Peng Wang, Quanxin Wang, Xinxin Wang, Yimin Wang, Yun Wang, Joshua D. Welch, Owen White, Elora Williams, Fangming Xie, Peng Xie, Feng Xiong, X. William Yang, Anna Marie Yanny, Zizhen Yao, Lulu Yin, Yang Yu, Jing Yuan, Hongkui Zeng, Kun Zhang, Meng Zhang, Zhuzhu Zhang, Sujun Zhao, Xuan Zhao, Jingtian Zhou, Xiaowei Zhuang, Brian Zingg. A multimodal cell census and atlas of the mammalian primary motor cortex. bioRxiv. https://doi.org/10.1101/2020.10.19.343129 \[[PDF](https://www.biorxiv.org/content/10.1101/2020.10.19.343129v1.full.pdf)\]

Alghamri MS, Avvari RP, Thalla R, Kamran N, Zhang L, Ventosa M, Taher A, Faisal SM, Núñez FJ, Fabiani MBG, Haase S, Carney S, Orringer D, Hervey-Jumper S, Heth J, Patil PG, Al-Holou WN, Eddy K, Merajver S, Ulintz PJ, Welch JD, Gao C, Liu J, Núñez G, Hambardzumyan D, Lowenstein PR, Castro M. G-CSF Secreted by Epigenetically Reprogrammed Mutant IDH1 Glioma Stem Cells Reverses the Meloid Cells'-Mediated Immunosuppressive Tumor Microenvironment. bioRxiv. https://doi.org/10.1101/2020.07.22.215954 \[[PDF](https://www.biorxiv.org/content/biorxiv/early/2020/07/24/2020.07.22.215954.full.pdf?%3Fcollection=)\]

Zizhen Yao, Hanqing Liu, Fangming Xie, Stephan Fischer, A. Sina Booeshaghi, Ricky S
Adkins, Andrew I. Aldridge, Seth A. Ament, Antonio Pinto-Duarte, Anna Bartlett, M. Margarita Behrens,
Koen Van den Berge, Darren Bertagnolli, Tommaso Biancalani, Héctor Corrada Bravo, Tamara Casper,
Carlo Colantuoni, Heather Creasy, Kirsten Crichton, Megan Crow, Nick Dee, Elizabeth L Dougherty,
Wayne I. Doyle, Sandrine Dudoit, Rongxin Fang, Victor Felix, Olivia Fong, Michelle Giglio, Jesse Gillis,
Jeff Goldy, Mike Hawrylycz, Hector Roux de Bézieux, Brian R. Herb, Ronna Hertzano, Xiaomeng Hou,
Qiwen Hu, Jonathan Crabtree, Jayaram Kancherla, Matthew Kroll, Kanan Lathia, Yang Eric Li, Jacinta
D. Lucero, Chongyuan Luo, Anup Mahurkar, Delissa McMillen, Naeem Nadaf, Joseph R. Nery,
Sheng-Yong Niu, Joshua Orvis, Julia K. Osteen, Thanh Pham, Olivier Poirion, Sebastian Preissl,
Elizabeth Purdom, Christine Rimorin, Davide Risso, Angeline C. Rivkin, Kimberly Smith, Kelly Street,
Josef Sulc, Thuc Nghi Nguyen, Michael Tieu, Amy Torkelson, Herman Tung, Eeshit Dhaval Vaishnav,
Valentine Svensson, Charles R. Vanderburg, Vasilis Ntranos, Cindy van Velthoven, Xinxin Wang, Owen
R. White, Z. Josh Huang, Peter V. Kharchenko, Lior Pachter, John Ngai, Aviv Regev, Bosiljka Tasic,
Joshua D. Welch, Joseph R. Ecker, Evan Z Macosko, Bing Ren, BRAIN Initiative Cell ensus Network
(BICCN), Hongkui Zeng, Eran A. Mukamel. An integrated transcriptomic and epigenomic atlas of mouse primary motor cortex cell types. bioRxiv. https://doi.org/10.1101/2020.02.29.970558 \[[PDF](https://www.biorxiv.org/content/biorxiv/early/2020/03/02/2020.02.29.970558.full.pdf)\]

Gao C, Preissl S, Luo C, Castanon R, Sandoval J, Rivkin A, Nery JR, Behrens MM, Ecker JR, Ren B, Welch JD. Iterative Refinement of Cellular Identity from Single-Cell Data Using Online Learning. bioRxiv. https://doi.org/10.1101/2020.01.16.909861. \[[PDF](https://www.biorxiv.org/content/10.1101/2020.01.16.909861v2.full.pdf)\]
Accepted for presentation at RECOMB 2020.

Liu J, Gao C, Sodicoff J, Kozareva V, Macosko EZ, Welch JD. Jointly defining cell types from multiple single-cell datasets using LIGER. Nat Protoc (2020). https://doi.org/10.1038/s41596-020-0391-8 \[[PDF](https://rdcu.be/b8pqb)\] \*Equal contribution

Matsushita Y, Nagata M, Kozloff K, Welch J, Mizuhashi K, Tokavanich N, Hallett S, Link D, Nagasawa T, Ono W, and Ono N. A Wnt-mediated transformation of the bone marrow stromal cell identity orchestrates skeletal regeneration. Nature Communications. 11, 332 (2020). \[[HTML](https://www.nature.com/articles/s41467-019-14029-w)\] \[[PDF](https://www.nature.com/articles/s41467-019-14029-w.pdf)\]

Zhou Y, Liu Z, Welch JD, Gao X, Wang L, Garbutt T, Keepers B, Ma H, Prins JF, Shen W, Liu J, Qian L. Single-Cell Transcriptomic Analyses of Cell Fate Transitions during Human Cardiac Reprogramming. Cell Stem Cell. 2019 Jun 12. pii: S1934-5909(19)30224-3. doi: 10.1016/j.stem.2019.05.020. \[[HTML](https://www.sciencedirect.com/science/article/pii/S1934590919302243)\]

Welch JD\*, Kozareva V, Ferreira A, Vanderburg C, Martin C, Macosko EZ\*. Single-cell multi-omic integration compares and contrasts features of brain cell identity. Cell. 2019 Jun 13;177(7):1873-1887.e17. doi: 10.1016/j.cell.2019.05.006. Epub 2019 Jun 6. \[[PDF](https://www.cell.com/action/showPdf?pii=S0092-8674%2819%2930504-5)\]  
\*Co-corresponding 

Rodriques SG\*, Stickels RR\*, Goeva A Martin CA, Murray E, Vanderburg CR, Welch JD, Chen LM, Chen F+,
Macosko EZ+. Slide-seq: A Scalable Technology for Measuring Genome-Wide Expression at High Spatial Resolution. Science 29 Mar 2019:
Vol. 363, Issue 6434, pp. 1463-1467 DOI: 10.1126/science.aaw1219 \[[PDF](http://science.sciencemag.org/content/sci/363/6434/1463.full.pdf)\]
\*Equal contribution
+Co-corresponding

Liu Z\*, Wang L\*, Welch JD\*, Ma H, Zhou Y, Vaseghi HR, Yu S, Wall JB, Alimohamadi S, Zheng M, Yin C, Shen W, Prins JF, Liu J, Qian L. Single-cell transcriptomics reconstructs fate conversion from fibroblast to cardiomyocyte. Nature (02 November 2017). 551, 100–104 doi:10.1038/nature24454 \[[PDF](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5954984/pdf/nihms908332.pdf)\]
\*Equal contribution

Welch JD, Hartemink AJ, Prins JF. MATCHER: manifold alignment reveals correspondence between single cell transcriptome and epigenome dynamics. Genome Biology 2017. 18:138. \[[PDF](https://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-017-1269-0)\]
*Also accepted at RECOMB 2017 under the title “E Pluribus Unum: United States of Single Cells”.

Lackey P, Welch JD, Marzluff WM. TUT7 Catalyzes the Uridylation of the 3’ End of Histone mRNA for Rapid Degradation. RNA 2016. doi:10.1261/rna.058107.116 \[[PDF](https://rnajournal.cshlp.org/content/22/11/1673.full.pdf+html)\]

Welch JD\*, Williams LA\*, DiSalvo M\*, Brandt AT, Marayati R, Sims CE, Allbritton NL, Prins JF, Yeh JJ, Jones CD. Selective Single Cell Isolation for Genomics Using Microraft Arrays. Nucleic Acids Research 2016. doi: 10.1093/nar/gkw700 \[[PDF](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5041489/pdf/gkw700.pdf)\]
*Equal contribution

Lyons SM, Cunningham CH, Welch JD, Groh B, Guo AY, Wei B, Whitfield ML, Xiong Y, Marzluff WF. A subset of replication-dependent histone mRNAs are expressed as polyadenylated RNAs in terminally differentiated tissues. Nucleic Acids Research 2016. doi: 10.1093/nar/gkw620 \[[PDF](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5100578/pdf/gkw620.pdf)\]

Welch JD, Hartemink AJ, Prins JF. SLICER: Inferring branched, nonlinear cellular trajectories from single cell RNA-seq data. Genome Biology 2016, 17:106  doi:10.1186/s13059-016-0975-3 \[[PDF](https://genomebiology.biomedcentral.com/track/pdf/10.1186/s13059-016-0975-3)\]
*Also accepted at RECOMB 2016 (doi: http://dx.doi.org/10.1101/047845)

Welch JD, Hu Y, Prins JF. Robust Detection of Alternative Splicing in a Population of Single Cells. Nucleic Acids Research 2016. doi: 10.1093/nar/gkv1525 \[[PDF](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4856971/pdf/gkv1525.pdf)\]
*Also accepted as a highlight at ISMB 2016

Brooks L, Lyons SM, Mahoney JM, Welch JD, Liu Z, Marzluff WM, Whitfield ML. A multi-protein occupancy map of the mRNP on the 3′ end of histone mRNAs. RNA 2015. \[[PDF](https://rnajournal.cshlp.org/content/21/11/1943.full.pdf+html)\]

Welch JD\*, Slevin MK\*, Tatomer D, Duronio RJ, Prins JF, Marzluff WF. EnD-Seq and AppEnD: Sequencing 3′ ends to identify nontemplated tails and degradation intermediates. RNA 2015. \[[PDF](https://rnajournal.cshlp.org/content/21/7/1375.full.pdf+html)\]
*Equal contribution

Welch JD, Baran-Gale J, Perou C, Sethupathy P, Prins JF. Pseudogenes Transcribed in Breast Invasive Carcinoma Show Subtype-Specific Expression and ceRNA Potential. BMC Genomics 2015. \[[PDF](https://bmcgenomics.biomedcentral.com/track/pdf/10.1186/s12864-015-1227-8)\]

Slevin MK, Meaux S, Welch JD, Bigler R, Miliani de Marval PL, Su W, Rhoads RE, Prins JF, Marzluff WF Deep Sequencing Shows Multiple Oligouridylations Are Required for 3′ to 5′ Degradation of Histone mRNAs on Polyribosomes. Molecular cell 53 (6), 1020-1030. March 2014. \[[PDF](https://www.cell.com/action/showPdf?pii=S1097-2765%2814%2900199-3)\]

Lichtenberg J, Kurz K, Liang X, Al-ouran R, Neiman L, Nau LJ, Welch JD, Jacox E, Bitterman T, Ecker K, Elnitski L, Drews F, Lee SS, Welch LR. WordSeeker: concurrent bioinformatics software for discovering genome-wide patterns and
word-based genomic signatures. BMC Bioinformatics. 2010 Dec 21;11 Suppl 12:S6.

Lichtenberg J, Yilmaz A, Welch JD, Kurz K, Liang X, Drews F, Ecker K, Lee SS, Geisler M, Grotewold E, Welch LR. The word landscape of the non-coding segments of the Arabidopsis thaliana genome. BMC Genomics. 2009 Oct 8;10:463.

Lichtenberg J, Jacox E, Welch JD, Kurz K, Liang X, Yang MQ, Drews F, Ecker K, Lee SS, Elnitski L, Welch LR. Word-based characterization of promoters involved in human DNA repair pathways. BMC Genomics. 2009 Jul 7;10 Suppl 1:S18.
